subxt/src/rpc/types.rs

```diff
@@ -239,30 +239,41 @@ pub type SystemProperties = serde_json::Map<String, serde_json::Value>;
 ///
 /// This is copied from `sp-transaction-pool` to avoid a dependency on that crate. Therefore it
 /// must be kept compatible with that type from the target substrate version.
+///
+/// Substrate produces `camelCase` events, while smoldot produces `CamelCase` events.
```

Comment:
Posting a new comment for visibility, because GitHub's UI isn't great.

Since this bug is fixed in smoldot, I'd suggest revert the changes you've made here.